xrpl/core/binarycodec/definitions/definitions.json

```diff
@@ -2178,43 +2479,45 @@
     "telCAN_NOT_QUEUE_FULL": -387,
 
     "temMALFORMED": -299,
-    "temBAD_AMOUNT": -298,
-    "temBAD_CURRENCY": -297,
-    "temBAD_EXPIRATION": -296,
-    "temBAD_FEE": -295,
-    "temBAD_ISSUER": -294,
-    "temBAD_LIMIT": -293,
-    "temBAD_OFFER": -292,
-    "temBAD_PATH": -291,
-    "temBAD_PATH_LOOP": -290,
-    "temBAD_REGKEY": -289,
-    "temBAD_SEND_XRP_LIMIT": -288,
-    "temBAD_SEND_XRP_MAX": -287,
-    "temBAD_SEND_XRP_NO_DIRECT": -286,
-    "temBAD_SEND_XRP_PARTIAL": -285,
-    "temBAD_SEND_XRP_PATHS": -284,
-    "temBAD_SEQUENCE": -283,
-    "temBAD_SIGNATURE": -282,
-    "temBAD_SRC_ACCOUNT": -281,
-    "temBAD_TRANSFER_RATE": -280,
-    "temDST_IS_SRC": -279,
-    "temDST_NEEDED": -278,
-    "temINVALID": -277,
-    "temINVALID_FLAG": -276,
-    "temREDUNDANT": -275,
-    "temRIPPLE_EMPTY": -274,
-    "temDISABLED": -273,
-    "temBAD_SIGNER": -272,
-    "temBAD_QUORUM": -271,
-    "temBAD_WEIGHT": -270,
-    "temBAD_TICK_SIZE": -269,
-    "temINVALID_ACCOUNT_ID": -268,
-    "temCANNOT_PREAUTH_SELF": -267,
-    "temINVALID_COUNT": -266,
-    "temUNCERTAIN": -265,
-    "temUNKNOWN": -264,
-    "temSEQ_AND_TICKET": -263,
-    "temBAD_NFTOKEN_TRANSFER_FEE": -262,
+    "temBAD_AMM_OPTIONS": -298,
+    "temBAD_AMM_TOKENS": -297,
+    "temBAD_AMOUNT": -296,
```

Comment:
Looks like there's some breaking changes in here...

Reply:
How are these breaking changes? All original fields were kept except their integer values changed.

Reply:
To clarify, I think this is a breaking change for rippled - does this not change how binary blobs will be deserialized, if one has one of the old numbers?

Reply:
@mvadari The AMM error codes have been relocated so now the current order is preserved.